pkg/network/netlink/consumer.go

```diff
@@ -370,10 +370,7 @@ func LoadNfConntrackKernelModule(ns netns.NsHandle) error {
 	if _, err = conn.Send(req); err != nil {
 		return fmt.Errorf("error while sending netlink request: %w", err)
 	}
-	_, _, err = sock.ReceiveAndDiscard()
-	if err != nil {
-		return fmt.Errorf("error while trying to load dummy entry from netlink: %w", err)
-	}
+	_, _, _ = sock.ReceiveAndDiscard()
```

Comment:
Do we want to send a non-junk request instead? 

I imagine some errors will indicate that conntrack didn't get loaded, and others would indicate that conntrack *did* get loaded but predictably didn't respond to our junk message.

Reply:
So the only non-junk request I've gotten through successfully in my local testing was a netlink dump. Dumps are much more costly performance wise and could trigger some timeouts if the netlink table on the customer host is large. There is probably a way to do an error-free single tuple lookup/get, I haven't been able to find how to do that though